cypress/support/cypress-support/src/utils.js

```diff
@@ -672,28 +672,72 @@ global.resolveDeviceVariable = function (key) {
  *
  */
 
-class FireLog {
+class FireLog extends Function {
   constructor() {
-    if (!FireLog.instance) {
-      FireLog.instance = this;
-    }
+    // Creating the function body dynamically
+    const functionBody = `
+      return function (...args) {
+        return this.log(...args);
+      }
+    `;
+    super('...args', functionBody);
+
+    const handler = {
+      apply: function (target, thisArg, argumentsList) {
+        let message;
+        const methodName = target.name;
+        if (target.hasOwnLog) {
+          // If the method has its own logging, just apply it
+          return Reflect.apply(target, thisArg, argumentsList);
+        } else {
+          if (argumentsList.length > 2)
+            message =
+              argumentsList[argumentsList.length - 1] +
+              ' expected ' +
```

Comment:
Can the log be `Expected: <Expected> Actual: <Actual>`

There were concerns previously about users having difficulty differentiating between the actual response and source of truth.

Reply:
fixed

Reply:
I don't see this change in the PR, please confirm if the commit was pushed

Reply:
yes, it wasn't commited, fixed it